crates/wasmtime/src/runtime/vm/cow.rs

```diff
@@ -406,7 +406,12 @@ impl MemoryImageSlot {
         tunables: &Tunables,
     ) -> Result<()> {
         assert!(!self.dirty);
-        assert!(initial_size_bytes <= self.static_size);
+        assert!(
+            initial_size_bytes <= self.static_size,
+            "initial_size_bytes <= self.static_size failed: \
```

Comment:
I think this line of the error message can be omitted as `assert!` should print out the expression that failed

Reply:
Unfortunately, if you specify any custom message, then it doesn't show the original expression anymore: https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=fb30232a93261b5bfa0bd219eac3ea7c